src/connections.js

```diff
@@ -16,12 +15,33 @@ const dappMetadata = {
   url: 'https://metamask.github.io/test-dapp/',
 };
 
+// eslint-disable-next-line require-unicode-regexp
+const isAndroid = /Android/i.test(navigator.userAgent);
+
 const sdk = new MetaMaskSDK({ dappMetadata });
 
-export const walletConnect = createWeb3Modal({
-  ethersConfig: defaultConfig({ metadata: dappMetadata }),
-  projectId: 'e6360eaee594162688065f1c70c863b7', // test id
-});
+export const initializeWeb3Modal = () => {
+  if (isAndroid) {
+    try {
+      // eslint-disable-next-line node/global-require
+      const { createWeb3Modal, defaultConfig } = require('@web3modal/ethers5');
+
+      const web3Modal = createWeb3Modal({
+        ethersConfig: defaultConfig({ metadata: dappMetadata }),
```

Comment:
we need the test projectId in order to work

`projectId: 'e6360eaee594162688065f1c70c863b7', // test id`